File: matlab/Environment.m
classdef Environment < handle
    
    properties
        cars
    end
    
    methods
        
        function this = Environment()
            this.cars = repmat(Car,1,0);
        end
        
        function [x] = get_next_station_following(this, car_position)
        	x = inf;
        end
        
    end

end

File: java/src/Environment.java
import java.util.ArrayList;
import java.util.List;

public class Environment {

    public List<Car> cars;

    public Environment() {
        this.cars = new ArrayList<Car>();
    }

    public double get_next_station_following(double car_position){
        return Float.POSITIVE_INFINITY;
    }

}
